app/workers/aggregation_data_mailer_worker.rb

```diff
@@ -1,5 +1,4 @@
 class AggregationDataMailerWorker
-  include Sidekiq::Worker
   include DumpWorker
```

Comment:
Why is the mailer including the DumpWorker? Couldn't we have a DumpMailerWorker module instead?